labellab_mobile/lib/model/label.dart

```diff
@@ -7,17 +7,20 @@ class Label {
   String id;
   String name;
   String type;
+  String mongoId;
```

Comment:
Can you replace mongo reference here? Variable of the client doesn't have to depict backends technologies. Refactor to a neutral name.

Reply:
Noted :+1: Thank you.